scripts/mysql_system_tables.sql

```diff
@@ -129,6 +129,16 @@ SET @create_innodb_index_stats="CREATE TABLE IF NOT EXISTS innodb_index_stats (
 	PRIMARY KEY (database_name, table_name, index_name, stat_name)
 ) ENGINE=INNODB DEFAULT CHARSET=utf8 COLLATE=utf8_bin STATS_PERSISTENT=0";
 
+SET @create_vtmd="CREATE TABLE IF NOT EXISTS vtmd (
+	trx_id_start			BIGINT UNSIGNED NOT NULL,
+	trx_id_end			BIGINT UNSIGNED NOT NULL,
+	old_name			VARCHAR(64),
+	name				VARCHAR(64) NOT NULL,
+	frm_image			BLOB,
+	col_renames			BLOB,
+	PRIMARY KEY(trx_id_start, trx_id_end, name)
```

Comment:
Don't need `trx_id_end` in PK. Read about [Multiple column index vs multiple indexes](https://www.percona.com/blog/2014/01/03/multiple-column-index-vs-multiple-indexes-with-mysql-56/) to know better on the topic.